client/src/index.css

```diff
@@ -0,0 +1,184 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@font-face {
+  font-family: "Poppins";
+  src: url("./assets/fonts/Poppins-Regular.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: "Rajdhani";
+  src: url("./assets/fonts/Rajdhani-Regular.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: "Nunito Sans";
+  src: url("./assets/fonts/NunitoSans_7pt-Regular.ttf") format("truetype");
+}
+
+:root {
+  --primaryColor: #e1b7ff;
+  --primaryLight: #e1b7ffb3;
+  --primaryDark: #b44dfe;
+  --secondaryColor: #eaf3f5;
+  --lightColor: #fdfdfd;
+  --darkColor: #353535;
+  --logoFont: "Poppins", sans-serif;
+  --headingFont: "Rajdhani", sans-serif;
+  --textFont: "Nunito Sans", sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: var(--textFont);
+  color: var(--darkColor);
+  background-color: var(--lightColor);
+}
+
+h1,
+h2,
+h3,
+h4 {
+  line-height: 120%;
+}
+
+h1 {
+  font-size: 2.5rem;
+  font-family: var(--headingFont);
+}
+
+h2 {
+  font-size: 2rem;
+  font-family: var(--headingFont);
+}
+
+h3 {
+  font-size: 1.5rem;
+  font-family: var(--headingFont);
+  font-weight: 600;
+}
+
+h4 {
+  font-size: 1.2rem;
+  font-family: var(--headingFont);
+  font-weight: 500;
+}
+
+header {
+  width: 100%;
+}
+
+section {
+  width: 100%;
+  padding: 1rem;
+}
+
+footer {
+  width: 100%;
+  padding: 0.5rem 1rem;
+}
+
+ul {
+  list-style-type: none;
+}
+
+li {
+  margin-left: 0.25rem;
+}
+
+button {
+  height: 2.5rem;
+  padding: 0.5rem 1rem;
+  text-align: center;
+  border-radius: 15px;
+  font-weight: 600;
+  background-color: var(--primaryColor);
+  outline: none;
```

Comment:
Disabling the outline property without managing focus disables the default keyboard navigation behavior to indicate that this element has taken focus. Either leave the default behavior, or manage to signal focus